docs/cimpl6LanguageReference.md

```diff
@@ -1039,9 +1026,9 @@ Each section in a Map file begins with a statement that establishes a **resource
 
 | CIMPL Class | FHIR Target | Example |
 |----------|---------|---------|
-| `Entry` | **resource** | _SocialHistoryObservation `maps to` **Observation**:_|
-| Entry | Profile | _LaboratoryObservation `maps to` **http://hl7.org/fhir/us/core/StructureDefinition/us-core-observationresults**:` |
-| `Group` | **complex data type** | _Timing `maps to` **Timing**:_ |
+| `Entry` | **resource** | `SocialHistoryObservation maps to Observation:` |
```

Comment:
If Observation is the name of a FHIR profile, it should be bolded, same for Timing.

Reply:
Correct as is. This is a CIMPL mapping statement, in CIMPL language.  All examples in CIMPL are written as code.